massa-async-pool/src/message.rs

```diff
@@ -212,8 +214,12 @@ impl AsyncMessage {
     /// Compute the ID of the message for use when choosing which operations to keep in priority (highest score) on pool overflow.
     /// For now, the formula is simply `score = (gas_price * max_gas, rev(emission_slot), rev(emission_index))`
     pub fn compute_id(&self) -> AsyncMessageId {
+        let denom = if self.max_gas > 0 { self.max_gas } else { 1 };
         (
-            std::cmp::Reverse(self.gas_price.saturating_mul_u64(self.max_gas)),
+            std::cmp::Reverse(
+                Amount::from_str(&Ratio::new(self.fee.to_raw(), denom).to_string())
```

Comment:
Please refrain from converting through string.
```suggestion
                Ratio::new(self.fee.to_raw(), denom)
```

Reply:
We want it to be a Reverse<Amount> right?

Reply:
yes, from the biggest to the smallest

Reply:
@gterzian ping